content/articles/time-series-analysis-and-forecasting-using-auto-time-series/index.md

```diff
@@ -0,0 +1,390 @@
+    alt: Time Series Analysis and Forecasting using Auto Time Series example image
+---
+A time series is a collection of data points organized in successive order over time. A time series shows all the data set variables that change over time.
+<!--more-->
+Time series analysis involves extracting meaningful patterns and other attributes of the historical data. This enables the model to gain knowledge about the trends in the dataset. Time series forecasting builds a model that predicts future values based on historical data. Times series models can be used to forecast forex exchange, business sales, stock prices, weather forecasts, and Covid-19 spread. 
+
+In stock prediction, a time series model tracks the movement of stock prices, such as Apple stock, it then identifies patterns and trends. This helps the model to forecast future Apple stock prices. Accurate prediction of the stock price will yield profit to the investors. In this tutorial, we will build an electricity consumption prediction model using Auto Time Series library(Auto-TS).
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [Getting started with Auto Time Series library](#getting-started-with-auto-time-series-library)
+- [Benefits of using Auto Time Series library](#benefits-of-using-auto-time-series-library)
+- [Installing Auto Time Series library](#installing-auto-time-series-library)
+- [Working with the dataset](#working-with-the-dataset)
+- [Loading the dataset](#loading-the-dataset)
+- [Plotting the line graph](#plotting-the-line-graph)
+- [Splitting the dataset](#splitting-the-dataset)
+- [Selecting the timestamp and the target columns](#selecting-the-timestamp-and-the-target-columns)
+- [Initializing the Auto Time Series model](#initializing-the-auto-time-series-model)
+- [Outputs](#outputs)
+- [Selecting the best model](#selecting-the-best-model)
+- [Actuals vs Forecast values](#actuals-vs-forecast-values)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+### Prerequisites
+To easily understand this article, a reader should:
+
+- Understand [time series](/engineering-education/introduction-to-time-series/)
+- Know how to build a [time series model](/engineering-education/sales-forecasting-with-prophet/)
+- Understand [time series decomposition in Python](/engineering-education/time-series-decomposition-in-python/)
+- Know some of the different [types of time series models](https://codeit.us/blog/machine-learning-time-series-forecasting)
+- Know [Google Colab notebook](https://colab.research.google.com/)
+
+### Getting started with Auto Time Series library
+Auto Time-series (Auto-TS) is an open-source Python library for automating time series analysis and forecasting. It is designed to rapidly train high-accuracy models within a short time. Auto-Ts automatically runs multiple time series models on the training dataset. It then automatically selects the best model from all the models.
```

Comment:
Is it Auto-TS or Auto-Ts? Please ensure there is consistency throughout our tutorial. In addition, add an external link to Auto Time Series documentation